src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -2586,6 +2603,11 @@ public int getCapacity() {
             .register(plugin);
         }
 
+        new SlimefunItem(categories.magicalResources, SlimefunItems.RAINBOW_LEATHER, RecipeType.ANCIENT_ALTAR,
+        new ItemStack[] {new ItemStack(Material.RABBIT_HIDE), new ItemStack(Material.LEATHER), new ItemStack(Material.RABBIT_HIDE), new ItemStack(Material.LEATHER), SlimefunItems.RAINBOW_RUNE, new ItemStack(Material.LEATHER), new ItemStack(Material.RABBIT_HIDE), new ItemStack(Material.LEATHER), new ItemStack(Material.RABBIT_HIDE)},
+        new SlimefunItemStack(SlimefunItems.RAINBOW_LEATHER, 8))
```

Comment:
I think this is way too cheap.

```suggestion
        new ItemStack[] {new ItemStack(Material.RABBIT_HIDE), new ItemStack(Material.EMERALD), new ItemStack(Material.RABBIT_HIDE), new ItemStack(Material.LEATHER), SlimefunItems.RAINBOW_RUNE, new ItemStack(Material.LEATHER), new ItemStack(Material.RABBIT_HIDE), new ItemStack(Material.LEATHER), new ItemStack(Material.RABBIT_HIDE)},
        new SlimefunItemStack(SlimefunItems.RAINBOW_LEATHER, 4))
```